lib/susedistribution.pm

```diff
@@ -605,6 +605,14 @@ sub init_consoles {
                         password => $testapi::password,
                         username => 'root'
                     });
```

Comment:
Maybe adding here comment that it's for zVM and JeOS? (the previous one is for zkvm). Or not needed because all developers can use `git blame`?